README.md

```diff
@@ -1 +1,4 @@
-This is a work in progress.
+# Shell commands using streams
+
+Port useful commands from the GNU `coreutils` to Haskell functions using
+streamly.
```

Comment:
We can add that the API of this package is unstable and may change frequently in next few releases.